src/components/Admins.vue

```diff
@@ -24,22 +24,25 @@
 		<span :class="{ hidden: !isLoading }" class="icon icon-loading" />
 		<div v-for="admin in admins" :key="admin.id" class="admin-NcAvatar-container">
 			<NcAvatar :user="admin.internal ? admin.id : null"
+				:title="admin.displayname"
 				:display-name="admin.displayname"
 				:size="64"
 				:is-no-user="!admin.internal"
 				:show-user-status="false" />
 			<NcActions v-if="!admin.internal">
-				<NcActionButton icon="icon-close" @click="deleteAdditionalAdmin(admin)">
-					{{ $t('privacy', 'Remove external admin') }}
-				</NcActionButton>
+				<NcActionButton icon="icon-close"
+					:aria-label="t('privacy', 'Remove external {propertyName} admin', { propertyName: admin.displayname })"
+					:title="t('privacy', 'Remove external {propertyName} admin', { propertyName: admin.displayname })"
+					@click="deleteAdditionalAdmin(admin)" />
```

Comment:
Hey, sorry, but why are we adding an aria label on a native button element. This does _not_ make any sense to me

Reply:
> By default, a button's accessible name is the content between the opening and closing [<button>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) tags, an image's accessible name is the content of its [alt](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-alt) attribute, and a form input's accessible name is the content of the associated [<label>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label) element.

Reply:
Ah, this is because `title` is not the content of the button.
This is an icon-only button! 

An aria label is actually necessary! :+1: :rocket:

Reply:
![Screenshot from 2023-01-26 12-17-07](https://user-images.githubusercontent.com/6078378/214822836-500f410b-e529-445f-bf13-342ef9e83a7d.png)


because this is focusable interactive element which have no text inside. Am i wrong?

Reply:
Nope, I was in the wrong!
I assumed the `title` prop was used for the button inner text as some other components also uses `title`.
But because there is indeed just an icon and no descriptive text inside the button, you're 100% right, an `aria label` is necessary :)